plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -667,17 +667,49 @@ global proc mayaUsdMenu_duplicateToUSD( string $proxy, string $obj )
     if (!hasPrimUpdater())
         return;
 
+    // Convert proxy to long form.
```

Comment:
What does a long form entail?

Reply:
Retrieves the full absolute name of the Maya object.